app/layout.tsx

```diff
@@ -25,7 +25,7 @@ export default function RootLayout({
   return (
     <html lang="en">
       <body
-        className={`${inter.variable} ${architects_daughter.variable} font-inter antialiased bg-gray-900 text-gray-200 tracking-tight`}
+        className={`${inter.variable} ${architects_daughter.variable} font-inter antialiased bg-white-100 text-secondary-main tracking-tight`}
```

Comment:
Are we going with background full white? I thought we're going just slightly beige
<img width="936" alt="Screenshot 2024-06-14 at 12 29 51 AM" src="https://github.com/trypear/pear-landing-page/assets/59063950/33badd1f-6b72-456a-a4b1-5d45a8989988">